spyder/plugins/editor/widgets/tests/test_introspection.py

```diff
@@ -25,6 +27,35 @@
 LOCATION = osp.realpath(osp.join(os.getcwd(), osp.dirname(__file__)))
 
 
+@pytest.mark.slow
+@pytest.mark.use_introspection
+@flaky(max_runs=3)
+@pytest.mark.skipif(os.name == 'nt' or not PY2,
+                    reason="Times out on AppVeyor and fails on PY3/PyQt 5.6")
+def test_calltip(lsp_codeeditor, qtbot):
+    """Test that the calltip in editor is hidden when matching ')' is found."""
```

Comment:
Let's change this docstring to

    Test that calltips are hidden when a matching ')' is found.